### DISCUSSION TITLE
使用 Obsidian Git 来定期同步简悦配置文件以及使用简悦的合辑功能整理标注到 Obsidian

### DISCUSSION THREAD

**Discussion by Dounm** [Kenshin/simpread#4593] [服务]
我个人的知识管理方案是
1. 使用 简悦 进行阅读标注
2. 使用 简悦稍后读的合辑功能 对标注进行初步整理
3. 最后定期把 合辑 的内容整理到 Obsidian

Obsidian 的同步方案是 Obsidian Git 同步到 Github，考虑到简悦的配置文件 `simpread_config.json` 也需要同步，因此就将其也放到 Obsidian Vault 文件夹里进行同步了。

具体配置如下：
1. 在 Obsidian Vault 的文件路径（我的是 `/Users/dounm/git/SecondBrain` ）下创建 `.simpread` 子文件夹，并将原有配置文件 `simpread_config.json` 拷贝过去

   <img width="587" alt="image" src="https://user-images.githubusercontent.com/9065640/195536717-edd7e693-2a97-4309-9b14-4ce337c1f3ba.png">

2. 配置 简悦同步助手 的 同步文件夹路径 如下

    <img width="700" alt="image" src="https://user-images.githubusercontent.com/9065640/195537225-0661e293-2c9c-4d0f-a584-bf5e40fea6ab.png">

3. 上述配置完毕后，就可以发现 `simpread_config.json` 已经被 Obsidian Git 所感知，其也会**按照配置定期与整个 Obsidian Vault 一块同步到 Github**

    <img width="834" alt="image" src="https://user-images.githubusercontent.com/9065640/195537776-33f1d158-ff83-495c-8762-8d44c7ed33ea.png">

**Comment by Kenshin** [comment]
# 后续

非常有趣的一套方案，这套方案最有意思的地方在于：

> 使用**简悦稍后读的合辑功能，对标注进行初步整理**，然后**定期把合辑的内容（手动）整理到 Obsidian** 。

大多数简悦用户都使用了当标注时自动导入到 Obsidain 的功能，而这套方案却「反其道行之」，使用手动的方式来整理，所以我询问了 @Dounm 为什么要这么做？

他的 [回复](https://github.com/Kenshin/simpread/issues/4592#issuecomment-1277284849) 如下：

> 其实就是离散的手动整理。每周末找个时间去把本周的稍后读里的标注整理到合辑，然后一个合辑里，如果对某些概念的阐释我觉着成熟了，就手动将其整理到 Obsidian 对应的概念里。
> 
> 相比于自动化同步到 Obsidian 的方案，我始终觉着手动虽然更慢，但整理的过程中理解的更深

**Comment by Kenshin** [comment]
# 使用 `git` 来同步简悦数据

[这套方案](https://github.com/Kenshin/simpread/discussions/4593) 另外一个有趣的地方在于：可以用 `git` 来同步简悦的全部数据。

## 一个例子

文中提到简悦配置文件位置 `/Users/dounm/git/SecondBrain/.simpread`

<img width="836" alt="image" src="https://user-images.githubusercontent.com/81074/195791987-c6616a42-eb60-40e0-ac12-20d0994be0ab.png">

所以简悦同步文件夹和导出文件夹都可以设置在这个目录下，如：

<img width="651" alt="image" src="https://user-images.githubusercontent.com/81074/195792527-24e29526-d5dd-4a83-8789-d131bdd92564.png">

这样使用 `git` 来管理简悦 + Obsidian 全部的数据。
